docs/flashbots-core/miners/advanced/mev-geth-introduction.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: mev-geth introduction
 ---
+
 ## MEV-Geth: a proof of concept
```

Comment:
```suggestion
### MEV-Geth: a proof of concept
```

Reply:
given this isn't a title anymore, makes sense to bring it back to same level as other titles imo